airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -183,6 +211,7 @@ def _group_logs_by_host(self, logs):
         grouped_logs = defaultdict(list)
         for log in logs:
             key = getattr_nested(log, self.host_field, None) or "default_host"
+            print(f"es_task_handler.py _group_logs_by_host...key = {key}")
```

Comment:
I guess this one should be either turned into debug log or removed?

Reply:
ahh Thanks. I just removed it.